bin/otomi

```diff
@@ -244,6 +244,9 @@ function execute() {
       docker_terminal_params='-ti'
       drun bash
       ;;
+    bats)
+      drun bats -T "$@" bin/tests
```

Comment:
`-T` also outputs the time it took for the script to run.

Reply:
nice!